### PR TITLE
test: Small improvements to running iOS tests locally

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): 

## PR Type

What kind of change does this PR introduce?

- Build or CI related changes

## What is the current behavior?

Running iOS tests locally for debugging purposes was a bit complicated.

## What is the new behavior?

- Added a group for specific class and method filter
- `IGNORE_RERUN_FILE` variable add to ensure the "failed test" file is ignored and the filtered tests always run

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: build/test-scripts/local-ios-uitest-run.sh [hunk 1]
```diff
@@ -1,16 +1,19 @@
 #!/bin/bash
+export UITEST_TEST_TIMEOUT=120000
 export BUILD_SOURCESDIRECTORY=`pwd`/../..
 export BUILD_ARTIFACTSTAGINGDIRECTORY=/tmp/uno-uitests-results
 export UITEST_SNAPSHOTS_ONLY=false
 export UITEST_SNAPSHOTS_GROUP=01
 export UNO_UITEST_IOSBUNDLE_PATH="$BUILD_SOURCESDIRECTORY/src/SamplesApp/SamplesApp.iOS/bin/iPhoneSimulator/Release/SamplesApp.app"
```
Comment (jeromelaban) [inline build/test-scripts/local-ios-uitest-run.sh:7]:
```suggestion
export BUILD_SOURCESDIRECTORY=`pwd`/../..
export BUILD_ARTIFACTSTAGINGDIRECTORY=/tmp/uno-uitests-results
export UITEST_SNAPSHOTS_ONLY=false
export UITEST_SNAPSHOTS_GROUP=01
export UNO_UITEST_IOSBUNDLE_PATH="$BUILD_SOURCESDIRECTORY/src/SamplesApp/SamplesApp.iOS/bin/iPhoneSimulator/Release/SamplesApp.app"
export UITEST_TEST_TIMEOUT=120000
```

File: build/test-scripts/ios-uitest-run.sh [hunk 2]
```diff
@@ -82,8 +89,7 @@ export UNO_TESTS_LOCAL_TESTS_FILE=$BUILD_SOURCESDIRECTORY/src/SamplesApp/Samples
 export UNO_UITEST_BENCHMARKS_PATH=$BUILD_ARTIFACTSTAGINGDIRECTORY/benchmarks/ios-automated
 export UNO_UITEST_RUNTIMETESTS_RESULTS_FILE_PATH=$BUILD_SOURCESDIRECTORY/build/RuntimeTestResults-ios-automated.xml
 
-if [ $(wc -l < "$UNO_TESTS_FAILED_LIST") -eq 1 ];
-then
+if [ $(wc -l < "$UNO_TESTS_FAILED_LIST") -eq 1 ] && [ "$IGNORE_RERUN_FILE" != "true" ]; then
```
Comment (jeromelaban) [inline build/test-scripts/ios-uitest-run.sh:92]: We could probably use default values like this for the `IGNORE_RERUN_FILE` variable:

```suggestion
IGNORE_RERUN_FILE="${IGNORE_RERUN_FILE:=false}"

if [ $(wc -l < "$UNO_TESTS_FAILED_LIST") -eq 1 ] && [ "$IGNORE_RERUN_FILE" != "true" ]; then
```

File: build/ci/.azure-devops-ios-tests.yml [hunk 3]
```diff
@@ -71,6 +71,7 @@ jobs:
     UITEST_AUTOMATED_GROUP: 1
     UITEST_TEST_TIMEOUT: '270000'
     UITEST_ALLOW_RERUN: 'true'
+    IGNORE_RERUN_FILE: 'false'
```
Comment (carldebilly) [inline build/ci/.azure-devops-ios-tests.yml:74]: Why it's not the default value?

**Review by carldebilly — APPROVED** [review]

File: build/test-scripts/local-ios-uitest-run.sh (hunk 1 above)
Comment (MartinZikmund) [reply build/test-scripts/local-ios-uitest-run.sh:7]: Reordered the variables 👍

File: src/Uno.UI-vs4mac-iOS-only.sln [hunk 4]
```diff
@@ -41,8 +41,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Xaml", "SourceGenerator
 EndProject
```
Comment (kazo0) [inline src/Uno.UI-vs4mac-iOS-only.sln:41]: These .sln changes are supposed to be here right?

Comment (MartinZikmund) [reply src/Uno.UI-vs4mac-iOS-only.sln:41]: Yes these are intentional, the sln was broken before

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**